src/Parser/BalanceDruid/Modules/Features/CastEfficiency.js

```diff
@@ -0,0 +1,74 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+
+import CoreCastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+
+class CastEfficiency extends CoreCastEfficiency {
+  static CPM_ABILITIES = [
+    ...CoreCastEfficiency.CPM_ABILITIES,
+    // Rotational Spells
+    {
+      spell: SPELLS.NEW_MOON,
+      category: CastEfficiency.SPELL_CATEGORIES.ROTATIONAL,
+      getCooldown: (haste, combatant) => {
+        const cd = (combatant.owner.fightDuration / 1000) / (((combatant.owner.fightDuration / 1000) - 2)/45);
+        return cd;
+      },
+      noSuggestion: true,
+    },
+    {
+      spell: SPELLS.HALF_MOON,
+      category: CastEfficiency.SPELL_CATEGORIES.ROTATIONAL,
+      getCooldown: (haste, combatant) => {
+        const cd = (combatant.owner.fightDuration / 1000) / (((combatant.owner.fightDuration / 1000) - 17)/45);
+        return cd;
+      },
+      noSuggestion: true,
+    },
+    {
+      spell: SPELLS.FULL_MOON,
+      category: CastEfficiency.SPELL_CATEGORIES.ROTATIONAL,
+      getCooldown: (haste, combatant) => {
+        const hasMooncloak = combatant.hasFinger(ITEMS.RADIANT_MOONLIGHT.id);
```

Comment:
@Iskalla Radiant Moonlight is a back right?

Reply:
Oops, legendary moon cloak, yes, sorry